hw/ip/uart/data/uart.hjson

```diff
@@ -262,7 +266,9 @@
       fields: [
         { bits: "0",
           name: "TXEN",
-          desc: "Enable TX pin override control"
+          desc: "Enable TX pin override control",
+          tags: [// writes to ovrd.txen causes tx output to be forced to ovrd.txval causing protocol violation
+                 "excl:CsrAllTests:CsrExclWrite"]
```

Comment:
OK. At first, I thought we could utilize `swaccess` and `hwaccess` to determine the `tags`. But this example shows it is not. :)